### ISSUE TITLE
Check citation scheme and refsDecl for /tlg0085/tlg001/tlg0085.tlg001.opp-grc3.xml

### ISSUE THREAD

**Issue by sonofmun** [OpenGreekAndLatin/First1KGreek#101]

**Comment by planatheisa** [comment]
This is a play.

**Comment by annettegessner** [comment]
As discussed: Just change the refsDecl according to #210 to just address lines and ignore all the other div-tags:
```xml
<cRefPattern n="line" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div//tei:l[@n='$1'])"/>
```

**Comment by jduff-chs** [comment]
Addressed in #247
